cloudferrylib/os/actions/check_cloud.py

```diff
@@ -12,13 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import time
+from cloudferrylib.base import exception
 from cloudferrylib.base.action import action
 from cloudferrylib.utils import utils as utl
 from keystoneclient.openstack.common.apiclient import exceptions as ks_exc
 from novaclient import exceptions as nova_exc
-from cloudferrylib.base import exception
 
 LOG = utl.get_log(__name__)
+LIMIT_RETRY_CHECK_INSTANCE = 25
```

Comment:
> Maybe it is better to use `retry` option from `migrate` section of config instead of hardcoded value?
> Current option `retry` is used for other targets
For what targets? I thought this is actually a target :)